day-1/ChronalCalibration/ChronalCalibration/Program.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;
+
+namespace ChronalCalibration
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var inputFilePath = @"C:\Users\joakim\repos\AoC-2018\day-1\ChronalCalibration\ChronalCalibration\Input.txt";
```

Comment:
This file path does not seem to work for me :( Maybe there is no Joakim on my machine!